arctic/chunkstore/chunkstore.py

```diff
@@ -363,8 +363,15 @@ def update(self, symbol, item, chunk_range=None):
             range and overwrite it with the data in item. This allows the user
             to update with data that might only be a subset of the
             original data.
+        upsert: True or False
+            if True, will write the data even if the symbol does not exist.
+            If true and an upsert is performed, chunk_range will be ignored
+        chunk_size: ?
```

Comment:
it depends on the chunker. for the date chunker, its an arctic DateRange object, or a Pandas DatetimeIndex